realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/UserTests.kt

```diff
@@ -412,9 +390,19 @@ class UserTests {
         user.logOut()
     }
 
-    // FIXME
-    @Ignore("Not implemented yet")
+    @Test
     fun isLoggedIn() {
+        var anonUser = app.login(Credentials.anonymous())
+        val user: User = app.registerUserAndLogin(TestHelper.getRandomEmail(), "123456")
+
+        assertTrue(anonUser.isLoggedIn)
+        assertTrue(user.isLoggedIn)
+
+        anonUser.logOut()
+        assertFalse(anonUser.isLoggedIn)
```

Comment:
Since testing the overlapping session explicitly I would say that you should include an
```
assertTrue(user.isLoggedIn)
```
here - After logging out one user, but before logging the other one out).